plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraConnectorTest.java

```diff
@@ -30,29 +30,24 @@
 import org.testng.SkipException;
 import org.testng.annotations.Test;
 
+import java.math.BigDecimal;
```

Comment:
Could you send a new PR for `TestCassandraConnectorTest` because it is unrelates to type mapping tests?

Reply:
Okay, I'll do that.

Reply:
Done.